apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources.properties

```diff
@@ -314,6 +314,7 @@ invalid.organization.name=Invalid Organization Name
 enter.code.sent.smsotp=Enter the code sent to your mobile phone
 auth.with.smsotp=Authenticating with SMS OTP
 error.code.expired.resend=The code entered is expired. Click Resend Code to continue.
+error.recaptcha.failed=The recaptcha verification failed. Please try again.
```

Comment:
Shall we add i18n strings for other languages as well listed in [identity-apps](https://github.com/wso2/identity-apps/tree/9122363f778fd9ad752af497613b63ce571cbeb2)/[apps](https://github.com/wso2/identity-apps/tree/9122363f778fd9ad752af497613b63ce571cbeb2/apps)/[authentication-portal](https://github.com/wso2/identity-apps/tree/9122363f778fd9ad752af497613b63ce571cbeb2/apps/authentication-portal)/[src](https://github.com/wso2/identity-apps/tree/9122363f778fd9ad752af497613b63ce571cbeb2/apps/authentication-portal/src)/[main](https://github.com/wso2/identity-apps/tree/9122363f778fd9ad752af497613b63ce571cbeb2/apps/authentication-portal/src/main)/[resources](https://github.com/wso2/identity-apps/tree/9122363f778fd9ad752af497613b63ce571cbeb2/apps/authentication-portal/src/main/resources)/[org](https://github.com/wso2/identity-apps/tree/9122363f778fd9ad752af497613b63ce571cbeb2/apps/authentication-portal/src/main/resources/org)/[wso2](https://github.com/wso2/identity-apps/tree/9122363f778fd9ad752af497613b63ce571cbeb2/apps/authentication-portal/src/main/resources/org/wso2)/[carbon](https://github.com/wso2/identity-apps/tree/9122363f778fd9ad752af497613b63ce571cbeb2/apps/authentication-portal/src/main/resources/org/wso2/carbon)/[identity](https://github.com/wso2/identity-apps/tree/9122363f778fd9ad752af497613b63ce571cbeb2/apps/authentication-portal/src/main/resources/org/wso2/carbon/identity)/[application](https://github.com/wso2/identity-apps/tree/9122363f778fd9ad752af497613b63ce571cbeb2/apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application)/[authentication](https://github.com/wso2/identity-apps/tree/9122363f778fd9ad752af497613b63ce571cbeb2/apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication)/[endpoint](https://github.com/wso2/identity-apps/tree/9122363f778fd9ad752af497613b63ce571cbeb2/apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint)
/i18n/